tests/utils/iotrace.py

```diff
@@ -54,10 +55,16 @@ def check_if_tracing_active(self) -> bool:
 
         :return: True if iotrace process found, False otherwise
         '''
+        pid = self.pid
         output = TestRun.executor.run('pgrep iotrace')
         if output.stdout == "":
+            TestRun.LOGGER.info("No iotrace processes found.")
             return False
         else:
+            self.pid = output.stdout
+            message = f"Iotrace process found with PID {self.pid}"
+            message += " but it's not searched process." if pid != self.pid else "."
+            TestRun.LOGGER.info(message)
```

Comment:
I would test if pids matches in test, not here.

Reply:
I refactored this a little. now it's more useful method.